src/commonTest/kotlin/fr/acinq/lightning/json/JsonTestsCommon.kt

```diff
@@ -15,4 +17,19 @@ class JsonTestsCommon : LightningTestSuite() {
         JsonSerializers.json.encodeToString(state)
     }
 
+    @Test
+    fun `serialize bolt11 invoice`() {
+        val invoice =
+            Bolt11Invoice.read("lntb123450n1pnx4cf2pp565tka26famjckm35lakwsrtnmfk7nzwm3va2u8tdu3kw7u80n5hqcqpjsp5guyuj6v84zyfxm3ae8y49rffgkcxsky73hun8mwvqfjdxw46ea0q9q7sqqqqqqqqqqqqqqqqqqqsqqqqqysgqdq523jhxapqd9h8vmmfvdjsmqz9gxqyjw5qrzjqwfn3p9278ttzzpe0e00uhyxhned3j5d9acqak5emwfpflp8z2cnfl6m5dzjwjw4hyqqqqlgqqqqqeqqjqedwgzyf2kuzyu4erj4xhdtknc9d8y8xkt8z80cpulg2q0mvgdcv4e7mgntf2nhur0x72k57ql7zx8ydzwtrxcnx9nk4pj65vfnhd3hsqyzlyxm")
+                .get()
+        JsonSerializers.json.encodeToString(JsonSerializers.Bolt11InvoiceSerializer, invoice)
```

Comment:
You encode to JSON but don't seem to compare with an expected result.

Reply:
True, but the goal here is only to verify that the serializer doesn't fail at runtime. Those serializers are only used for information and adding hard non-reg would be overkill I think.